core/storage/user/gae_models.py

```diff
@@ -33,6 +33,8 @@ class UserSettingsModel(base_models.BaseModel):
 
     Instances of this class are keyed by the user id.
     """
+    # User id used to identify user by GAE.
```

Comment:
Add more context to explain that this is currently optional, and a TODO (+ issue) to remove/update this and change to required=True once the migration job is run.

Reply:
Done. #7659